graveler/graveler.go

```diff
@@ -388,14 +388,21 @@ type RefManager interface {
 	Log(ctx context.Context, repositoryID RepositoryID, commitID CommitID) (CommitIterator, error)
 }
 
+// Tree abstracts the data structure of the committed data.
```

Comment:
Will this interface evolve?  Otherwise, how do I use `GetTree`?  Is it just to check whether it `!= nil`, and make `IsTreeExist` seem like it returns a tree?

Reply:
The latter more or less for now. It will probably add the tree parts later on.